### PR TITLE
Refactor to include percentage stats

### PR DESCRIPTION
**Pull request by dlmccoy**
Screenshot 2019-10-09 00 44 49

Hi, I'd like to accomplish two things:
1. Display in the CLI a few of the percent coverages
2. Expose a more programatic interface for some of the stats metrics in a `Calculator` class so that I can use the values for some other data visualization over time

### CODE REVIEW

**Comment by dlmccoy** [comment]
@jaredbeck any feedback on this?

File: README.md [hunk 1]
```diff
@@ -25,14 +25,21 @@ Or install it yourself:
...
+# total_classes    	112433
+
+# Progress for file coverage
+# sigil_ignore     	12	    0.20 %
+# sigil_false      	5466	    91.60 %
+# sigil_true       	460	    7.71 %
+# sigil_strict     	12	    0.20 %
+# sigil_strong     	17	    0.28 %
+# ---------------------------------------
+# Total: 		5967	100%
```
Comment (jaredbeck) [inline README.md:42]: yay! percentages are a good idea

File: lib/sorbet_progress/version.rb [hunk 2]
```diff
@@ -4,6 +4,6 @@
 # :nodoc:
 module SorbetProgress
   def self.gem_version
-    Gem::Version.new("0.2.3")
+    Gem::Version.new("0.2.4")
```
Comment (jaredbeck) [inline lib/sorbet_progress/version.rb:7]: This is fine for now, but in the future, please don't change gem version

**Review by jaredbeck — APPROVED** [review]

**Comment by jaredbeck** [comment]
Released as 0.2.4. Thanks Dillon.

**Pull request merged**